internal/db/schema/migrations/postgres/2/90_auth.up.sql

```diff
@@ -0,0 +1,87 @@
+begin;
+
+-- By adding the name column to the base auth method type, the database can
+-- ensure that auth method names are unique across all sub types.
+alter table auth_method
+  add column name wt_name;
+
+alter table auth_method
+  add constraint auth_method_scope_id_name_uq
+    unique (scope_id, name);
+
+
+-- insert_auth_method_subtype() is a replacement of the function definition in
```

Comment:
Should this migration also drop the name column from the subtype so that we don't have duplicate data?

Reply:
I don't think so, `name` is similar to the rest of the columns that are inserted into the base type's table.